lib/tasks/add_navigation_links.rake

```diff
@@ -59,56 +103,56 @@ namespace :navigation_links do
       name: "Code of Conduct",
       icon: thumb_up_icon,
       display_only_when_signed_in: false,
-      position: 5,
+      position: 0,
```

Comment:
moving the code of conduct to the top of the other section was intentional and called out by the design doc. :+1: